Code/Framework/AzCore/AzCore/Settings/SettingsRegistryMergeUtils.h

```diff
@@ -260,7 +275,8 @@ namespace AZ::SettingsRegistryMergeUtils
     //!     example: --regdump /My/Array/With/Objects
     //! --regdumpall Dumps the entire settings registry to output.
     //! Note that this function is only called in development builds and is compiled out in release builds.
-    void MergeSettingsToRegistry_CommandLine(SettingsRegistryInterface& registry, AZ::CommandLine commandLine, bool executeCommands);
+    void MergeSettingsToRegistry_CommandLine(SettingsRegistryInterface& registry, AZ::CommandLine commandLine,
```

Comment:
Could you update the function docs to replace `executeCommands` with `m_parseRegdumpCommands`?

Reply:
Yes, I'll fix that as well